examples/v3.rs

```diff
@@ -3,16 +3,13 @@ use std::collections::HashMap;
 use sendgrid::v3::*;
 
 fn main() {
-    let mut cool_header = HashMap::new();
+    let mut cool_header = HashMap::with_capacity(2);
```

Comment:
Regarding the changes to `with_capacity` here and below, is there any advantage to having such a small capacity? I was under the impression that `new` has a reasonable default.

Reply:
I believe that rusts allocs every multiple of 2, so there's one allocation on first push to a size of 1, and another to a size 2.  Therefore there's one less allocation by using `with_capacity(2)` rather than `new`. 

Moreover, there's no downside to using with_capacity 🙂